Tests/ASNetworkImageNodeTests.mm

```diff
@@ -109,7 +109,6 @@ @implementation ASTestImageCache
 
 - (void)cachedImageWithURL:(NSURL *)URL callbackQueue:(dispatch_queue_t)callbackQueue completion:(ASImageCacherCompletion)completion
 {
-  ASDisplayNodeAssert(callbackQueue == dispatch_get_main_queue(), @"ASTestImageCache expects main queue for callback.");
```

Comment:
@Adlai-Holler do you remember why this was here?

Reply:
Nope, we can remove.